core/connection/interface.go

```diff
@@ -18,21 +18,27 @@
 package connection
 
 import (
-	"github.com/mysteriumnetwork/go-openvpn/openvpn"
-	"github.com/mysteriumnetwork/go-openvpn/openvpn/middlewares/state"
-
 	"github.com/mysteriumnetwork/node/communication"
 	"github.com/mysteriumnetwork/node/identity"
 	"github.com/mysteriumnetwork/node/service_discovery/dto"
-	"github.com/mysteriumnetwork/node/session"
 )
 
 // DialogCreator creates new dialog between consumer and provider, using given contact information
 type DialogCreator func(consumerID, providerID identity.Identity, contact dto.Contact) (communication.Dialog, error)
 
-// VpnClientCreator creates new vpn client by given session,
-// consumer identity, provider identity and uses state callback to report state changes
-type VpnClientCreator func(session.SessionDto, identity.Identity, identity.Identity, state.Callback, ConnectOptions) (openvpn.Process, error)
+// Connection represents a connection
+type Connection interface {
+	Start() error
```

Comment:
Extra line

Reply:
not sure what you mean by that, what extra line?